src/Features/Core/Portable/ReplacePropertyWithMethods/AbstractReplacePropertyWithMethodsService.cs

```diff
@@ -247,6 +259,13 @@ public void Do()
                 }
             }
 
+            private void ReplaceCref()
+            {
+                _editor.ReplaceNode(
+                    _cref,
+                    GetCrefReference(_cref));
+            }
```

Comment:
this method seems unnecessary (very simple, called in just one location).  Can you just inline?